src/mobile-token/utils.ts

```diff
@@ -58,5 +62,29 @@ export const getSdkErrorHandlingStrategy = (
  */
 export const getSdkErrorTokenIds = (err: any): string[] =>
   err instanceof TokenFactoryError
-    ? [err.pendingTokenId, err.activatedTokenId].filter(isDefined)
+    ? [err.pendingTokenId?.tokenId, err.activatedTokenId?.tokenId].filter(
+        isDefined,
+      )
     : [];
+
+/**
+ * from https://github.com/entur/abt-mobile-client-sdk/pull/368
+ */
+export const parseBffCallErrors = (error: any) => {
+  if (
+    error instanceof TypeError &&
+    error.message === 'Network request failed'
```

Comment:
I'm not really happy about using free text messages for this check. Take a look into whether there is some axios helpers that can check if the request failed because of network.